arangod/Pregel/Conductor.h

```diff
@@ -46,14 +47,49 @@
 #include "Pregel/Conductor/StoringState.h"
 #include "Pregel/Status/ConductorStatus.h"
 #include "Pregel/Status/ExecutionStatus.h"
+#include "Pregel/WorkerInterface.h"
 #include "velocypack/Builder.h"
 
 #include <chrono>
+#include <cstdint>
 #include <memory>
 
 namespace arangodb {
 namespace pregel {
 
+struct ClusterConductorWorker : NewIWorker {
+  ClusterConductorWorker(ExecutionNumber executionNumber, ServerID serverId,
+                         TRI_vocbase_t& vocbase)
+      : _executionNumber{std::move(executionNumber)},
+        _serverId{std::move(serverId)},
+        _vocbaseGuard{vocbase} {}
+  [[nodiscard]] auto loadGraph(LoadGraph const& graph)
+      -> futures::Future<ResultT<GraphLoaded>> override;
+
+ private:
+  ExecutionNumber _executionNumber;
+  ServerID _serverId;
+  const DatabaseGuard _vocbaseGuard;
+};
+
+struct SingleServerConductorWorker : NewIWorker {
```

Comment:
Please, explain in a short comment what this class is for: it's an API of a worker whose instances run on conductors in a single server.